src/docfx/cli/Program.cs

```diff
@@ -84,8 +83,9 @@ private static (string command, string docset, CommandLineOptions options) Parse
             ArgumentSyntax.Parse(args, syntax =>
             {
                 // Restore command
-                // usage: docfx restore [docset]
+                // usage: docfx restore [docset] [-t/--token token]
                 syntax.DefineCommand("restore", ref command, "Restores dependencies before build.");
+                syntax.DefineOption("t|token", ref options.Token, "The token used to restore dependency repositories");
```

Comment:
consider `--gitToken`, we could have other token types, also remove `t|`, it does not deserve a short hand.